### PR TITLE
Daniellecrobinson patch 1

### PR DESCRIPTION
**Pull request by joehand**

### CODE REVIEW

File: content/index.txt [hunk 1]
```diff
@@ -4,14 +4,54 @@ view: home
...
 ----
-description: We are a 501(c)(3) nonprofit supporting open source projects and open collaboration in public interest technology. We envision a future of technology where value-driven communities have tools that reflect their values. By connecting people across domains and building shared resources, we are working towards that future.
+description: We are a US-based 501(c)(3) nonprofit supporting open collaboration in public interest technology through fiscal sponsorship, collaborative projects in open infrastructure, and consulting. We envision a future where communities build technology infrastructure that reflect their values.
 ----
 background: /assets/background.jpg
 
 ----
 
+programs:
+communities:
```
Comment (joehand) [inline content/index.txt]: indents matter here. needs to have 2 spaces indent after programs.

File: content/index.txt [hunk 2]
```diff
@@ -4,14 +4,54 @@ view: home
...
+title: Collaborative Communities
+description: We develop [open resources to support the public interest technology community](https://osaos.codeforscience.org/) and advocate for open infrastructure.  We also consult to facilitate collaborations between organizations and provide strategic support to community-driven initiatives engaging with technology. Every quarter we host [a community call](https://www.notion.so/community-call) to bring people, projects, and external speakers together to discuss the open public interest technology ecosystem.
+
+projects:
```
Comment (joehand) [inline content/index.txt:24]: need separator `---` (see below)

Comment (joehand) [reply content/index.txt:24]: also indents

File: content/index.txt (hunk 1 above)
Comment (daniellecrobinson) [reply content/index.txt]: k

**Pull request merged**